htdocs/fourn/card.php

```diff
@@ -514,7 +514,7 @@
 	if (!empty($conf->product->enabled) || !empty($conf->service->enabled)) {
 		$langs->load("products");
 		//Query from product/liste.php
-		$sql = 'SELECT p.rowid, p.ref, p.label, p.fk_product_type, p.entity, p.tosell as status, p.tobuy as status_buy, p.tobatch as status_batch,';
+		$sql = 'SELECT distinct(p.rowid), p.ref, p.label, p.fk_product_type, p.entity, p.tosell as status, p.tobuy as status_buy, p.tobatch as status_batch,';
```

Comment:
We want all the references of supplier. We may have several references for same product.
If product A is sold in qty 10 with reference  A10  and sold in qty 100 with reference A100, we must have 2 lines reported for same product.